src/borg/key.py

```diff
@@ -258,8 +254,8 @@ def id_hash(self, data):
         return sha256(data).digest()
 
     def encrypt(self, chunk):
-        chunk = self.compress(chunk)
```

Comment:
the idea here was that chunks carry data and metadata and that both flow together through the whole processing pipeline. so don't strip away the metadata even if we do not use it here right now.